bundles/org.openhab.binding.opensprinkler/README.md

```diff
@@ -61,22 +61,25 @@ NOTE: Some channels will only show up if the hardware has the required sensor an
 
 | Channel Type ID | Item Type              |    | Description                                                                        |
 |-----------------|------------------------|----|------------------------------------------------------------------------------------|
-| rainsensor      | Switch                 | RO | This channel indicates whether rain is detected by the device or not.              |
-| sensor2         | Switch                 | RO | This channel is for the second sensor (if your hardware supports it).              |
+| cloudConnected  | Switch                 | RO | If the device is fully connected to the OpenSprinkler cloud this will show as 'ON'.|
```

Comment:
I am asking myself if a channel is really the good option.
The connection status should probably be something defining the thing status ?

Reply:
@jlaur : WDYT ?

Reply:
I think I need some more context. This binding a declared as `local`, so if I understand this correctly, this is just some information from the device whether it is also integrated with a cloud service in addition to being integrated through the local API? In that case, what would be the use of this channel? Perhaps it would be sufficient to provide this as a property?

Reply:
I believe a property is wrong as it has a state that changes. Read only channel seemed to fit. Your thoughts are correct and I also do not see the use case, but was easy to add it when doing other channels instead of someone requesting it later. It could possibly be used to fault find why you can not use their cloud to connect and control the sprinkler. The binding is full local and it is perhaps strange that the device can do both without trying to force you to do one over the other.

Reply:
I would prefer to delete the channel then to spend any more time on changing or adding extra lines to add it to things that are missing the channel. I don't have a lot of spare time these days.